Description: Change the dhcpd3 plugin to reflect debian spesific locations
 for the dhcpd configuration file and leases file
Debian bug: #543979
Author: Tom Feiner
--- munin-1.2.6.orig/node/node.d/dhcpd3.in	2005-03-29 20:32:59.000000000 +0000
+++ munin-1.2.6/node/node.d/dhcpd3.in	2009-08-27 21:04:57.873424331 +0000
@@ -48,8 +48,8 @@
 my %ips      = ();
 my $DEBUG    = 0;
 
-my $LEASEFILE  = $ENV{leasefile}  || "/var/lib/dhcp/dhcpd.leases";
-my $CONFIGFILE = $ENV{configfile} || "/etc/dhcpd.conf";
+my $LEASEFILE  = $ENV{leasefile}  || "/var/lib/dhcp3/dhcpd.leases";
+my $CONFIGFILE = $ENV{configfile} || "/etc/dhcp3/dhcpd.conf";
 
 if($ARGV[0] and $ARGV[0] eq "autoconf" ) {
     if($ret) {
